Makefile

```diff
@@ -50,6 +50,10 @@ docker-dev:
 	$(eval VAULT_TOKEN = $(shell docker logs grant-vault 2>&1 | grep "Root Token" | tail -1 | cut -d ' ' -f 3 ))
 	VAULT_TOKEN=$(VAULT_TOKEN) docker-compose -f docker-compose.yml -f docker-compose.dev.yml run --rm dev /bin/bash
 
+docker-cory-dev:
```

Comment:
Wondering what we should call this @husobee 

For context this allows for easy restarting of the server when there's a change in the code. 

I run the following watch command and when there's a file change the docker container automatically restarts and applies the saved file change.
```
fswatch . | xargs -I {} sh -c 'clear && echo "Rebuilding and restarting server" &&  $(docker ps -f "name=cory" --format "docker restart {{.ID}}")'
```

Reply:
https://github.com/brave-intl/bat-go/pull/327 addresses this